docs/troubleshooting.md

```diff
@@ -0,0 +1,11 @@
+# Troubleshooting 
```

Comment:
@justinsb I was thinking that all troubleshooting topics would eventually find their way into the same document.